src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -150,6 +120,7 @@ public int stem(char s[], int len) {
 
   /**
    * Re-writes the char array based on an exception.
+   * Used in case a stem is not a substring of the original string.
```

Comment:
@cmantas Since we're having "protected words" for leaving the word as is, can we safely remove this now? The only reason I can think of is converting a string to a same length string but different, or even worse expanding the string to a larger one. I don't think they are plausible scenarios though.